packages/squiggle-lang/src/rescript/Distributions/PointSetDist/PointSetDist_Scoring.res

```diff
@@ -1,46 +1,185 @@
-module KLDivergence = {
-  let logFn = Js.Math.log // base e
-  let integrand = (predictionElement: float, answerElement: float): result<
+type t = PointSetTypes.pointSetDist
```

Comment:
I don't think this should be called ``t`` here. t is really meant when it's the main type of the file, but here that isn't obvious. (It's not a ``Score`` type, and there's Scalar logic here)